Formula/adb-enhanced.rb

```diff
@@ -0,0 +1,39 @@
+class AdbEnhanced < Formula
+  include Language::Python::Virtualenv
+
+  desc "Swiss-army knife for Android testing and development"
+  homepage "https://ashishb.net/tech/introducing-adb-enhanced-a-swiss-army-knife-for-android-development/"
+  url "https://files.pythonhosted.org/packages/0b/87/b1375ade9b02c67d4a8fe760d9d8cc4250c37e1db19d0ba1343a3bc79d9c/adb-enhanced-2.5.21.tar.gz"
+  sha256 "976db27d1334782965592d7cd2202d4de803eecb1360f2c710ad07dcc2af3539"
+  license "Apache-2.0"
+
+  depends_on "python@3.11"
+
+  resource "docopt" do
+    url "https://files.pythonhosted.org/packages/a2/55/8f8cab2afd404cf578136ef2cc5dfb50baa1761b68c9da1fb1e4eed343c9/docopt-0.6.2.tar.gz"
+    sha256 "49b3a825280bd66b3aa83585ef59c4a8c82f2c8a522dbe754a8bc8d08c85c491"
+  end
+
+  resource "psutil" do
+    url "https://files.pythonhosted.org/packages/d6/0f/96b7309212a926c1448366e9ce69b081ea79d63265bde33f11cc9cfc2c07/psutil-5.9.5.tar.gz"
+    sha256 "5410638e4df39c54d957fc51ce03048acd8e6d60abc0f5107af51e5fb566eb3c"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    system "#{bin}/adbe", "--version"
+    system "#{bin}/adbe", "--help"
+    begin
+      system("#{bin}/adbe devices", exception: false)
+    rescue
+      # Expected failure
+      return
+    else
+      # Fail if no exception happens
+      assert false
+    end
```

Comment:
I dont quite this test, that is why I rewrite the test using `shell_output`

Reply:
This is failing with an exception?
What do you recommend on how to test this?

Reply:
`shell_output` takes an optional second parameter for the expected return code.

Reply:
Changed.

There seems to be very sparse documentation on `shell_output`, so, I was not able to find this earlier.